index.html

```diff
@@ -0,0 +1,104 @@
+<!doctype html>
+<html class="no-js" lang="">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title> Delicious | THE BEST FOOD BLOG ON THE WEB </title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+    <!-- Place favicon.ico in the root directory -->
+
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+    <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+  </head>
+  <body>
+
+    <nav>
+      <img class="banner-logo" src="http://i.imgur.com/5tFwpoN.png">
+      <h1> Delicious <br> THE BEST FOOD BLOG ON THE WEB </h1>
+      <ul>
+        <li> <img src="http://i.imgur.com/Xz0logE.png"> </li>
+        <li> <img src="http://i.imgur.com/FnlhJVH.png"> </li>
+        <li> <img src="http://i.imgur.com/pfmEjMI.png"> </li>
+        <li> <img src="http://i.imgur.com/ylgYf41.png"> </li>
+        <li> <img src="http://i.imgur.com/zSxL2IW.png"> </li>
+        <li> <img src="http://i.imgur.com/vCWEjwG.png"> </li>
+        <li> <img src="http://i.imgur.com/wxiFpvM.png"> </li>
+        <li> <img src="http://i.imgur.com/jrfNOkO.png"> </li>
+      <ul>
+    </nav>
+
+    <article>
+      <section class="instructions">
+        <h1> Chocolate Pizza </h1>
+        <p> POSTED ON <time datetime="2013-12-15"> 15 DEC 2013 </time> / DESSERTS </p>
+        <ul>
+          <li> <img src="http://i.imgur.com/mtX28lM.png"> </li>
+          <li> PRINT </li>
+        </ul>
+        <img class="food-pic" src="http://i.imgur.com/fB4svJW.png">
+
+        <p> For the fig-swirl: Melt butter over medium heat in a saucepan. Add brown sugar and stir to dissolve. Halve all of the figs and toss in the saucepan with water and lemon juice. Cook over medium heat, stirring frequently, until you have a chunky-jammy mixture. Add salt with one or two stires, set aside and let cool completely. </p>
+
+        <p> Ice cream: In a small pot over medium head, combine milk, and granulated sugar until sugar is completely dissolved and the milk is just barely lukewarm. Whisk in the egg yolks. Set mixture in the fridge and wait until the fig mixture is cooled. </p>
+
+        <p> Using an ice cream machine, pour liquids into the frozen basin and process according to the manufacturer instructions, i.e., let spin and thicken for 20 minutes before adding mascarpone, fig jam mixture, and the nuts. Continue to process for +/- 10 minutes. Pour semi-frozen mixture into a pyrex dish or glass tupperware. Freeze for at least two hours before serving. </p>
+      </section>
+      <section class="ingredients">
+        <img src="http://i.imgur.com/Tbl1pWm.png"
```

Comment:
Not closed. Also, this will probably just be a background-image declared through css on the parent container (ie the section class="ingredients"
Also may want to make the class slightly more specific